### PR TITLE
User menu + integrated accelerator if available

### PR DESCRIPTION
**Pull request by nymkappa**

### CODE REVIEW

**Review by wiz — APPROVED** [review]
Tested ACK @ v3.0.0-dev [2ff3e4acb]

**Comment by wiz** [comment]
Found one minor issue on mobile with the bottom sticky nav floating in the air for a while, would be nice to fix this before merging

**Review by wiz — CHANGES_REQUESTED** [review]
Tested NACK @ v3.0.0-dev [401004734]

The sponsor photos don't load now:

```
Failed to load resource /api/v1/about-page: the server responded with a status of 404 ()
```

File: frontend/src/app/components/master-page/master-page.component.html [hunk 1]
```diff
@@ -1,6 +1,14 @@
+<app-menu *ngIf="servicesEnabled" (loggedOut)="onLoggedOut()"></app-menu>
+
 <ng-container *ngIf="{ val: network$ | async } as network">
-<header *ngIf="headerVisible">
+<header *ngIf="headerVisible" style="position: fixed; width: 100%; z-index: 100;">
```
Comment (softsimon) [inline frontend/src/app/components/master-page/master-page.component.html:4]: Is the `headerVisible` @Input being used or can it be deleted?

**Comment by softsimon** [comment]
The left menu should auto-close if you click somewhere on the page outside of the menu.

File: frontend/src/app/components/master-page/master-page.component.html (hunk 1 above)
Comment (nymkappa) [reply frontend/src/app/components/master-page/master-page.component.html:4]: It's used in /sponsor and /enterprise

**Review by wiz — CHANGES_REQUESTED** [review]
Menu should not be on top of the upper-left logos or hamburger button, it should be under them, like YouTube

Screenshot 2023-08-22 at 13 24 13
Screenshot 2023-08-22 at 13 22 24

**Review by softsimon — DISMISSED** [review]
`officialMempoolSpace && stateService.env.ACCELERATOR` is no longer present to display the My Account button in the global footer, so it's always visible.

image
The offline-tag has shifted position.

**Review by wiz — APPROVED** [review]

**Review by wiz — CHANGES_REQUESTED** [review]
# 1) mempool icon on about page has moved
Screenshot 2023-08-29 at 19 47 10 (3)
Screenshot 2023-08-29 at 19 47 13 (3)

# 2) Footer is not sticky at bottom
Screenshot 2023-08-29 at 19 45 26 (3)
Screenshot 2023-08-29 at 19 45 20 (3)

**Pull request merged**